bunq/sdk/http/api_client.py

```diff
@@ -50,13 +50,13 @@ class ApiClient(object):
 
     # Default header values
     _USER_AGENT_BUNQ = 'bunq-sdk-python/1.10.16'
-    _GEOLOCATION_ZERO = '0 0 0 0 NL'
-    _LANGUAGE_EN_US = 'en_US'
-    _REGION_NL_NL = 'nl_NL'
-    _CACHE_CONTROL_NONE = 'no-cache'
+    GEOLOCATION_ZERO = '0 0 0 0 NL'
+    LANGUAGE_EN_US = 'en_US'
+    REGION_NL_NL = 'nl_NL'
+    CACHE_CONTROL_NONE = 'no-cache'
 
     # Request method names
-    _METHOD_POST = 'POST'
+    METHOD_POST = 'POST'
```

Comment:
perhaps factor these out into an Enum that's entirely public?

Reply:
For consistency, I will make the other 3 method names public as well.

Reply:
For now, I'd prefer to not go to far out of scope with this MR. Before creating an enum for these, I'd first like to properly see what the other SDKs do for similar constants.